libmusic/include/lmtypes.h

```diff
@@ -145,22 +159,48 @@ typedef struct Chord {
                 ((c1.__mRootNote == c2.__mRootNote) && (c1.__mQuality < c2.__mQuality)));
     }
 
-    friend std::ostream& operator<<(std::ostream& os, const Chord &c)
+    std::string toHarte() const
     {
-        os << c.__mRootNote << c.__mQuality;
-        if (c.__mBassNote != note_Unknown)
-            os << "/" << c.__mBassNote;
-        return os;
+        std::ostringstream ss;
+
+        ss << __mRootNote;
+        if (__mRootNote != note_Unknown) {
+            ss << ":" << __mQuality;
+        }
+        if (__mBassInterval > 1) {
+            ss << "/" << (int)__mBassInterval;
+        }
+
+        return ss.str();
     }
 
-    std::string toString()
+    std::string toUser() const
     {
         std::ostringstream ss;
+
         ss << __mRootNote << __mQuality;
-        if (__mBassNote != note_Unknown)
+        if (__mBassInterval > 1) {
```

Comment:
```suggestion
        if (__mBassInterval >= 1) {
```

so that we distinguish `C` and `C/C` templates ?

Reply:
let's not confuse `ChordTpl` and `Chord` classes. Please check my comment on this above: https://github.com/vmkononenko/music-dsp/pull/17#discussion_r615373243